### ISSUE TITLE
"using with evidence (an annotation id) #### is invalid, since there is no information about annotation id ####"

### ISSUE THREAD

**Issue by dustine32** [pantherdb/fullgo_paint_update#21]
A type of warning from PAINT validation service:

> PTN000218930 annotated to term GO:0000981 for annotation id 313863640 using with evidence (an annotation id) 385055487 is invalid, since there is no information about annotation id 385055487

I'm wondering if it's simply caused by an aggregate view switcheroo in the warning query. It looks like this should be valid. I can find annotation_id 313863640 in the paint_aggregate view and evidence (as annotation_id) 385055487 in go_aggregate view. None of these records or their "_evidence" records are obsoleted.

The warning query may be checking these two annotation_ids (one for paint, the other for go) against only one of the views? Otherwise, I can't find anything in the DB that would indicate this to be invalid. We'd need to double-check the service code.

**Comment by dustine32** [comment]
Likely caused by not factoring in experimental confidence codes in determining when to obsolete paint_evidence records during the update. Fix will be committed soon.

**Comment by pgaudet** [comment]
@dustine32 This is very cryptic. Let me know if there anything that needs testing or just looking at. 

Thanks, Pascale

**Comment by dustine32** [comment]
@pgaudet Agreed. @mugitty and I are just going through a list of warnings produced by her service for certain books. The example here is for PTHR11849. I think I've already found where in the update pipeline this can be fixed.

**Comment by dustine32** [comment]
We can test that this book is fixed with this URL:
http://panthercuratest.usc.edu/webservices/family.jsp?searchValue=PTHR11849&searchType=SEARCH_TYPE_AGG_FAMILY_ANNOTATION_INFO

It shouldn't return any warnings.
